crowdfunding/users/serializers.py

```diff
@@ -0,0 +1,24 @@
+from rest_framework import serializers
+from .models import CustomUser
+
+class CustomUserSerialiser(serializers.ModelSerializer):
+
+    class Meta:
+        model = CustomUser
+        fields = ['email', 'username', 'password']
+
+    def create(self, validated_data):
+        user = CustomUser(
+            email=validated_data['email'],
+            username=validated_data['username']
+        )
+        user.set_password(validated_data['password'])
+        user.save()
+        return user
+
+    def update(self, instance, validated_data):
+        instance.username = validated_data.get('username', instance.username)
+        instance.email = validated_data.get('email', instance.email)
+        instance.set_password = validated_data.get('password')
```

Comment:
set_password is a method, not a field, this might not behave in the way you were expecting!